internal/controller/biossettingsset_controller.go

```diff
@@ -239,6 +248,35 @@ func (r *BIOSSettingsSetReconciler) deleteOrphanBIOSSettings(
 	return errors.Join(errs...)
 }
 
+func (r *BIOSSettingsSetReconciler) updateSpec(
```

Comment:
Can we call this method `patchOrCreateBIOSSettingsfromTemplate` and only pass the template + the set list?